File: Common/Entity/src/main/java/it/ey/dto/Sezione32DTO.java
package it.ey.dto;

import lombok.*;
import lombok.experimental.SuperBuilder;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
public class Sezione32DTO extends SezioneBaseDTO
{
    private Long id;

    private Long idPiao;

    private String statoSezione;

    private String descrizioneContestoRiferimento;

    private String descrizioneObiettiviLavoroAgile;

    private String descrizioneStatoAttuazione;

    private String descrizioneFattoriAbilitanti;

    private String descrizionePersonaleAgile;

    private String descrizioneGiornateLavorate;

    private String descrizioneLivelloSoddisfazione;

    private String descrizioneContributi;

    private String descrizioneImpatti;

    private List<TabellaFunzionaleDTO> tabelleFunzionali;
}

File: dfp-piao-fe/src/app/shared/models/classes/sezione-32-dto.ts
import { TabellaFunzionaleDTO } from './tabella-funzionale-dto';

export class Sezione32DTO {
  id?: number;
  idPiao?: number;
  statoSezione?: string;
  descrizioneContestoRiferimento?: string;
  descrizioneObiettiviLavoroAgile?: string;
  descrizioneStatoAttuazione?: string;
  descrizioneFattoriAbilitanti?: string;
  descrizionePersonaleAgile?: string;
  descrizioneGiornateLavorate?: string;
  descrizioneLivelloSoddisfazione?: string;
  descrizioneContributi?: string;
  descrizioneImpatti?: string;

  tabelleFunzionali?: TabellaFunzionaleDTO[];

  campiModificati?: string;
  testoSezione?: string;
}
